tests/test_watchlists.py

```diff
@@ -111,6 +139,93 @@ def test_get_page_when_default_params_returns_expected_data(
         "/v1/watchlists", method="GET", query_string=urlencode(query)
     ).respond_with_json(data)
 
+
+@pytest.fixture
```

Comment:
Refactored a lot of the API calls in this testing module to be a `pytest.fixture` so that they could be used by both the SDK and CLI tests.  Thats responsible for most of the diff in the file thats not the CLI tests themselves.

Reply:
I like it